src/lib.rs

```diff
@@ -888,8 +888,8 @@ impl Connection {
             sent_count: 0,
 
             rx_data: 0,
-            max_rx_data: max_rx_data as usize,
-            max_rx_data_next: max_rx_data as usize,
+            max_rx_data: max_rx_data as u64,
```

Comment:
The two values above should already be `u64` so you don't need to cast them I think.
```suggestion
            max_rx_data,
```